protoc-gen-java-snowpea/README.md

```diff
@@ -0,0 +1,83 @@
+# Protoc Plugin: protoc-gen-java-snowpea
+
+This plugin generates tests for Google Event Types Library for Java to ensure the ability to unmarshal data.
+
+## Things to Know
```

Comment:
Great job documenting the code generation process! I definitely missed this in the Go repo and will make a point of adding something.